app/konnect/org-management/troubleshoot.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Troubleshoot Account and Org Issues
+no_version: true
+---
+
+## Identity Provider Errors
+
+### Groups claim is empty when Okta is the IdP
+
+#### Problem
+In the token payload, the `groups` claim is configured correctly but still
+appears empty, or it includes some groups but not all.
+
+#### Solution
+This issue might happen if the authorization server is pulling in additional
+groups from third-party applications (for example, Google groups).
+
+An Okta administrator needs to duplicate the third-party groups
+and re-create them directly in Okta. They can do this by exporting the group
+in CSV format, then importing the CSV file into Okta to populate the new group.
+
+### Named cookie not present
+
+#### Problem
+If the issuer URI is incorrect or incomplete, you may get the following error
```

Comment:
```suggestion
If the issuer URI is incorrect or incomplete, you will get the following error
```

Reply:
I think "may" is correct here, (or possibly change to `may likely`) as I'm sure Okta will change that message at some point in the future